modules/sdk-coin-btc/src/lib/keyPair.ts

```diff
@@ -0,0 +1,96 @@
+import { randomBytes } from 'crypto';
+import * as bip32 from 'bip32';
+import {
+  AddressFormat as BaseAddressFormat,
+  DefaultKeys,
+  isPrivateKey,
+  isPublicKey,
+  isSeed,
+  KeyPairOptions,
+  NotImplementedError,
+  Secp256k1ExtendedKeyPair,
+} from '@bitgo/sdk-core';
+import { address, crypto, networks } from 'bitcoinjs-lib';
+
+const DEFAULT_SEED_SIZE_BYTES = 16;
+
+// TODO: extend the AddressFormat enum in @bitgo/sdk-core with these values
+enum ExtendedAddressFormat {
+  mainnetBech32 = 'mainnetBech32',
+  testnetBech32 = 'testnetBech32',
+}
+
+export type AddressFormatType = BaseAddressFormat | ExtendedAddressFormat;
+export const AddressFormat = { ...BaseAddressFormat, ...ExtendedAddressFormat };
+
+/** Bitcoin key management. */
+export class KeyPair extends Secp256k1ExtendedKeyPair {
```

Comment:
Is this something that would be good to put in `abstract-utxo` and share among utxo coins?

Reply:
Absolutely, however, due to the dependency between modules I opted for having the first implemented directly in a class I can use. I'm moving the common code to the base class in a follow-up PR so subsequent UTXO coins only have to set up the network.

There are a few changes coming in `sdk-core` and `abstract-utxo` classes.

Also, the enum `AddressFormat` seems to be mixing the format and network. I think we may want to have them separated into 2 different parameters like:

```
export enum AddressFormat {
  hex = 'hex',
  base58 = 'base58',
  bech32 = 'bech32',
  bech32 = 'bech32',
}

export enum Network {
  mainnet = 'mainnet',
  testnet = 'testnet',
}
```

Then we can do something like:
`getAddress({ format: AddressFormat, network: Network })`, or even better, we can initialize the KeyPair class with the right network like `new KeyPair({ network: Network })`. Thoughts?